From 79beffbc118349ecc5ae6a030a29f3f481d06516 Mon Sep 17 00:00:00 2001
From: Valera <valera@example.com>
Date: Fri, 18 Sep 2026 22:45:08 +0300
Subject: [PATCH 1/3] lisa: Adjust display cutout and status bar height to
 100px

Align status bar height and cutout bounding box directly to the bottom
edge of the camera punch hole (center Y=68 + radius 32 = 100px), eliminating
excessive vertical padding.

Change-Id: I4ad443c2bea7d76b534e401fac48933db1b6fe47
---
 rro_overlays/FrameworkOverlayLisa/res/values/config.xml | 3 +--
 rro_overlays/FrameworkOverlayLisa/res/values/dimens.xml | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/rro_overlays/FrameworkOverlayLisa/res/values/config.xml b/rro_overlays/FrameworkOverlayLisa/res/values/config.xml
index 4faef3f..a4c6d33 100644
--- a/rro_overlays/FrameworkOverlayLisa/res/values/config.xml
+++ b/rro_overlays/FrameworkOverlayLisa/res/values/config.xml
@@ -334,8 +334,7 @@
          appearance of the cutout as much as possible, and may be smaller than
          config_mainBuiltInDisplayCutout
          -->
-    <string name="config_mainBuiltInDisplayCutoutRectApproximation" translatable="false">M 0,0 H 139 V 136 H 0 Z @left</string>
-
+    <string name="config_mainBuiltInDisplayCutoutRectApproximation" translatable="false">M 0,0 H 139 V 100 H 0 Z @left</string>
     <!-- Whether the display cutout region of the main built-in display should be forced to
          black in software (to avoid aliasing or emulate a cutout that is not physically existent).
          -->
diff --git a/rro_overlays/FrameworkOverlayLisa/res/values/dimens.xml b/rro_overlays/FrameworkOverlayLisa/res/values/dimens.xml
index 5ea4f6e..501673d 100644
--- a/rro_overlays/FrameworkOverlayLisa/res/values/dimens.xml
+++ b/rro_overlays/FrameworkOverlayLisa/res/values/dimens.xml
@@ -11,7 +11,7 @@
     <!-- Height of the status bar in portrait.
          Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
          -->
-    <dimen name="status_bar_height_portrait">108px</dimen>
+    <dimen name="status_bar_height_portrait">100px</dimen>
 
     <!-- Height of the status bar in landscape.
          Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-- 
2.55.0

